index.html

```diff
@@ -283,7 +291,8 @@ <h1>Contact me</h1>
                 <input type="text" name="name" placeholder="Enter name" maxlength="30" required>
                 <input type="email" name="email" placeholder="Enter email" required>
                 <textarea name="message" placeholder="Write your message here" maxlength="500" required></textarea>
-                <button type="submit" class="buttonaction">Get In Touch</button>
+                <div class="btnn">
+                <button type="submit" class="buttonaction">Get In Touch</button></div>
             </form>
```

Comment:
- [ ] Everything is good, there are small changes you need to make to the contact form please implement it. In your contact form, the small background is missing please implement it.
- [ ] I would suggest please decrease the width of form to match the design
### Your design
![your](https://user-images.githubusercontent.com/81727988/207806005-84f9ee9d-9245-4be0-b5a7-d3a125985237.png)

### Fimga design
![figaaaaaaaaaaaa](https://user-images.githubusercontent.com/81727988/207806019-96e65c19-2e69-43b6-be10-5fbb54f43e50.png)

Reply:
Please review the new request before I submit it again.
We have **Limited Request review...**